src/windows/footpath.c

```diff
@@ -358,6 +368,9 @@ static void window_footpath_dropdown()
 				continue;
 			if (pathType->flags & flags)
 				continue;
+			// Skip queue lines of scenario editor-only paths (only applicable when the game is in sandbox mode)
+			if(widgetIndex == WIDX_QUEUELINE_TYPE && pathType->flags & 4)
```

Comment:
This must have been an oversight on my part, since I normally always do this. I'll fix it.